sts-mock/src/testUtils/getMockSigningKey.ts

```diff
@@ -0,0 +1,16 @@
+import { SigningKey } from "../token/keyRetriever/keyRetriever";
+import { createPrivateKey } from "node:crypto";
+
+export function getMockSigningKey(): SigningKey {
+  const jwk = {
+    kty: "EC",
+    x: "X5Sk_TNlp2cZ_mUQ7p_Gqth7IVkLGaMFfUH-e6pzUC8",
+    y: "Iw5VG-5Sb-c954Xqanov47QhQVLEw3H9BdGNwR9RvOw",
+    crv: "P-256",
+    d: "eswmWYW980vOl7x0MZBqzAkxN215eogA0fFJCu6umfw",
+    kid: "iyVpkshZ0QKq5ORWz7mc76x0dAKUp4RS113tiHACjpQ",
+  };
+  const privateKey = createPrivateKey({ key: jwk, format: "jwk" });
+  const keyId = jwk.kid;
+  return { signingKey: privateKey, keyId };
```

Comment:
```suggestion
  const signingKey = createPrivateKey({ key: jwk, format: "jwk" });
  const keyId = jwk.kid;
  return { signingKey, keyId };
```

Thoughts? Just to avoid using two different terms for the same thing

Reply:
Or rename the interface to expect `privateKey` if you prefer that variable name

Reply:
Good idea, done!